src/Request/Request.php

```diff
@@ -182,7 +180,8 @@ private function send($uri, $method, array $arg_params = [])
             $params['json'] = $params['form_params'];
             unset($params['form_params']);
         }
-        $params[RequestOptions::VERIFY] = (strpos($host, 'onebox') === false);
+        $verify = (boolean)$this->getConfig()->get('verify_host_cert', true);
```

Comment:
Does this mean if I want to target a Onebox, I'll need to add `TERMINUS_VERIFY_HOST_CERT=false`?